client/javascript/Router.jsx

```diff
@@ -25,6 +25,12 @@ import Error404 from './pages/Error404';
 
 export default (store) => {
   const history = syncHistoryWithStore(browserHistory, store);
+  /* eslint-disable no-unused-vars */
+  history.listen((location) => {
+    if (window.appInsights) {
```

Comment:
I'm guessing this fires after the render is complete and the new title is in place?